src/scythe-bidder/constants.ts

```diff
@@ -18,11 +18,53 @@ export const factions = [
   "Rusviet",
 ] as const;
 
+export const MATS_BASE = [
+  "Industrial",
+  "Engineering",
+  "Patriotic",
+  "Mechanical",
+  "Agricultural",
+] as const;
+
+export const FACTIONS_BASE = [
+  "Crimea",
+  "Saxony",
+  "Polania",
+  "Nordic",
+  "Rusviet",
+] as const;
+
+export const MATS_IFA = [
+  "Industrial",
+  "Engineering",
+  "Militant",
+  "Patriotic",
+  "Innovative",
+  "Mechanical",
+  "Agricultural",
+] as const;
+
+export const FACTIONS_IFA = [
+  "Togawa",
+  "Crimea",
+  "Saxony",
+  "Polania",
+  "Albion",
+  "Nordic",
+  "Rusviet",
+] as const;
+
 export const SCYTHE_BIDDER = "scythe-bidder";
 
+export const SCYTHE_BASE = false;
+export const SCYTHE_IFA = true;
```

Comment:
I think these aren't needed, I think I find `const [isIfaActive, setIsIfaActive] = React.useState<boolean>(true);` and simply using that boolean instead of comparing to these more readable.

Reply:
And I think the default should be IFA on because that's what people are currently used to.

Reply:
Agreed and removed.

Reply:
And IFA set to ON.